src/Microsoft.DotNet.Build.Tasks.Packaging/src/NuGetPack.cs

```diff
@@ -243,11 +301,60 @@ public void Pack(string nuspecPath, Func<string, string> nuspecPropertyProvider,
             }
             catch (Exception e)
             {
-                string packageType = packSymbols ? "symbol" : "lib";
+                string packageType = "lib";
+                if (creatingSymbolsPackage)
+                {
+                    packageType = "symbol";
+                }
+                else if (packSymbols)
+                {
+                    packageType = "packed";
+                }
                 Log.LogError($"Error when creating nuget {packageType} package from {nuspecPath}. {e}");
             }
         }
 
+        private Manifest TransformManifestToPackedPackageManifest(Manifest manifest)
```

Comment:
OK I guess I didn't realize we were going to need both symbol and transport for CoreCLR if that is the case then I'm OK with the general idea.